tests/php/Migration/Version034000Date20260428144801Test.php

```
@@ -0,0 +1,232 @@
+	private const TEST_APP_TWO = 'phpunit_migration_botapp2';
+	private const TALK_BOT_ROUTE_PREFIX = 'talk_bot_route_';
+
+	private IDBConnection $db;
+	private ICrypto $crypto;
+	private Version034000Date20260428144801 $migration;
+
+	protected function setUp(): void {
+		parent::setUp();
+		$this->db = Server::get(IDBConnection::class);
+		$this->crypto = Server::get(ICrypto::class);
+		$this->migration = new Version034000Date20260428144801(
```

PHPStan (2.1.54)

[error] Class OCA\AppAPI\Migration\Version034000Date20260428144801 constructor invoked with 3 parameters, 2 required.


(arguments.count)